nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -425,6 +425,8 @@
 - Minimal installer ISOs are no longer built on the small channel.
   Please obtain installer images from the full release channels.
 
+- `openssl` now defaults to the latest version line `3.3.x`, instead of `3.0.x` before. While there should be no major code imcompatibilities, newer OpenSSL versions typically strengthen the default security level. This means, that you may have to explicitely allow older/weak ciphers, hashes and key lengths again. See: [OpenSSL security level documentation](https://docs.openssl.org/1.1.1/man3/SSL_CTX_set_security_level/).
```

Comment:
```suggestion
- `openssl` now defaults to the latest version line `3.3.x`, instead of `3.0.x` before. While there should be no major code incompatibilities, newer OpenSSL versions typically strengthen the default security level. This means that you may have to explicitly allow weak ciphers, hashes and key lengths if necessary. See: [OpenSSL security level documentation](https://docs.openssl.org/3.3/man3/SSL_CTX_set_security_level/).
```

Perhaps we should also mention that we’ll be tracking the latest release branch from here on out and that you shouldn’t expect the old versions to stick around?